akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -624,12 +624,19 @@ private final class Sdk(
           protocolMinorVersion = BuildInfo.protocolMinorVersion)
 
       override def reportError(err: UserFunctionError): Future[Done] = {
-        val severityString = err.severity.name.take(1) + err.severity.name.drop(1).toLowerCase(Locale.ROOT)
+        val severityString = err.severity match {
+          case Level.ERROR => "Error"
+          case Level.WARN  => "Warning"
```

Comment:
Warn => Warning was the reason I changed this, but seems better to do a full match than the letter conversions